WordPress/UITests/Tests/StatsTests.swift

```diff
@@ -6,8 +6,8 @@ class StatsTests: XCTestCase {
 
     override func setUpWithError() throws {
         setUpTestSuite()
-        _ = try LoginFlow.login(siteUrl: WPUITestCredentials.testWPcomSiteAddress, email: WPUITestCredentials.testWPcomUserEmail, password: WPUITestCredentials.testWPcomPassword)
-        statsScreen = try MySiteScreen()
```

Comment:
With the test no longer using the stored `statsScreen`, we should be able to remove the property from the class, too. See line 5 of this file.